core/models.py

```diff
@@ -1,3 +1,4 @@
+from typing import Optional
```

Comment:
I think we might want to run a linter on commit too, this doesn't seem to be used

Reply:
A spell checker would also be nice lol